src/renderer/bootstrap.tsx

```diff
@@ -19,7 +20,8 @@ type AppComponent = React.ComponentType & {
 export {
   React,
   Mobx,
-  LensExtensions
+  LensExtensions,
```

Comment:
Why do we need to export it here? Where/how it's used afterward?

Reply:
`LensExtensions` includes store related APIs which are needed also in renderer.

Reply:
Actually I mean whole `export` from `bootstrap.tsx`

Reply:
All extension apis are exported from `src/main/index.ts` (for main extension api) and `src/renderer/bootstrap.tsx` (for renderer extension api) using webpack (webpack allows to export modules from an entrypoint). Extensions can consume these via webpack externals.

Reply:
Hmm. I thought using/marking webpack's externals as global variable should be enough..
Is it documented somewhere in webpack's docs that entry point should export actually something in addition?

Reply:
https://webpack.js.org/configuration/output/#outputlibrarytarget

Reply:
Ok, thanks!